egs/heroico/s5/local/prepare_dict.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -u
+
+# Copyright 2017 John Morgan
+# Apache 2.0.
+
+set -o errexit
+
+[ -f ./path.sh ] && . ./path.sh
+
+wget http://www.openslr.org/resources/34/santiago.tar.gz
```

Comment:
I think it should be better 
```
if [ ! -f ./santiago.tar.gz ];  then
  wget ...
fi
```
You could also look at the wget's `-O ` switch -- probably it could be used to download the file directly into the destination directory.
Also, I'm mildly leaning towards downloading the file in run.sh, not here.